modules/cluster-logging-release-notes-5.3.z.adoc

```diff
@@ -27,11 +70,11 @@ This release includes link:https://access.redhat.com/errata/RHSA-2022:0044[RHSA-
 
 [id="openshift-logging-5-3-2-bug-fixes"]
 == Bug fixes
-* Before this update, Elasticsearch rejected logs from the Event Router due to a parsing error. This update changes the data model to resolve the parsing error. However, as a result, previous indices might cause warnings or errors within Kibana. The `kubernetes.event.metadata.resourceVersion` field causes errors until existing indices are removed or reindexed. If this field is not used in Kibana, you can ignore the error messages. If you have a retention policy that deletes old indices, the policy eventually removes the old indices and stops the error messages. Otherwise, manually reindex to stop the error messages. link:https://issues.redhat.com/browse/LOG-2087[LOG-2087])
+* Before this update, Elasticsearch rejected logs from the Event Router due to a parsing error. This update changes the data model to resolve the parsing error. However, as a result, previous indices might cause warnings or errors within Kibana. The `kubernetes.event.metadata.resourceVersion` field causes errors until existing indices are removed or reindexed. If this field is not used in Kibana, you can ignore the error messages. If you have a retention policy that deletes old indices, the policy eventually removes the old indices and stops the error messages. Otherwise, manually reindex to stop the error messages. (link:https://issues.redhat.com/browse/LOG-2087[LOG-2087])
 
 * Before this update, the OpenShift Logging Dashboard displayed the wrong pod namespace in the table that displays top producing and collected containers over the last 24 hours. With this update, the OpenShift Logging Dashboard displays the correct pod namespace. (link:https://issues.redhat.com/browse/LOG-2051[LOG-2051])
 
-* Before this update, any situation with Elasticsearch OutputDefault defined for cluster logforwarding and Elasticsearch outputs did not have structured keys, the configuration was replacing the output secret with the default secret used to communicate to the default log store. With this update, the defined output secret is correctly used.(link:https://issues.redhat.com/browse/LOG-2046[LOG-2046])
+* Before this update, any situation with Elasticsearch OutputDefault defined for cluster logforwarding and Elasticsearch outputs did not have structured keys, the configuration was replacing the output secret with the default secret used to communicate to the default log store. With this update, the defined output secret is correctly used. (link:https://issues.redhat.com/browse/LOG-2046[LOG-2046])
```

Comment:
```suggestion
* Before this update, if `outputDefaults.elasticsearch.structuredTypeKey` in the `ClusterLogForwarder` custom resource (CR) instance did not have a structured key, the CR replaced the output secret with the default secret used to communicate to the default log store. With this update, the defined output secret is correctly used. (link:https://issues.redhat.com/browse/LOG-2046[LOG-2046])
```

Reply:
This is my best guess at what this release note might mean. I recommend having an SME/QE review and approve it. Thanks.